malasakit-django/pcari/templates/render-question.html

```diff
@@ -0,0 +1,36 @@
+{% load i18n %}
+{% load static %}
+
+{% ifequal q.input_type 'range' %}
+  <span class="leftlabel">{% trans q.left_text %}</span>
+  <input id="rating-{{ q.id }}" type='range' value="{{ q.min_val }}"
+         min="{{ q.min_val }}" max="{{ q.max_val }}">
+  <span class="rightlabel">{% trans q.right_text %}</span>
+{% endifequal %}
+
+{% ifequal q.input_type 'number' %}
+  <div class="center">
+    <span class="leftlabel">{% trans q.left_text %}</span>
+    <input id="rating-{{ q.id }}" type='number' style="max-width:200px;"
+         min="{{ q.min_val }}" max="{{ q.max_val }}">
+    <span class="rightlabel">{% trans q.right_text %}</span>
+  </div>
+{% endifequal %}
+
+{% ifequal q.input_type 'dropdown' %}
+  <select id="rating-{{ q.id }}">
+    {% for option in q.options %}
+      <option>{% trans option %}</option>
+    {% endfor %}
+  </select>
+{% endifequal %}
+
+{% ifequal q.input_type 'bubble' %}
+  <div id="rating-{{ q.id }}" class="container">
+    {% for option in q.options %}
+      <input type="radio" name="rating-{{ q.id }}" value={{ q.option }}>
+      {% trans option %}
+      <br>
+    {% endfor %}
+  </div>
+{% endifequal %}
```

Comment:
This file should be broken up into four separate files, one for each component, so that we can use [template composition](https://docs.djangoproject.com/en/1.11/ref/templates/builtins/#include).